media-source-respec.html

```diff
@@ -824,16 +865,26 @@ <h4><dfn>Attaching to a media element</dfn></h4>
                       <li>[=Queue a task=] to [=fire an event=] named {{sourceopen}} at the <a>MediaSource</a>.</li>
                     </ol></dd>
                   </dl>
-                <li>Continue the <a def-id="resource-fetch-algorithm"></a> by running the remaining <a def-id="Otherwise-mode-is-local"></a> steps, with these clarifications:
+                <li>Continue the [=resource fetch algorithm=] by running the remaining
+                  <a def-id="Otherwise-mode-is-local"></a> steps, with these clarifications:
```

Comment:
I think they're requirements not clarifications.

Reply:
Sounds good to me. I'll change this to requirements.

Reply:
Done in 8ffdbb2